website/src/pages/Blog/index.tsx

```diff
@@ -337,6 +338,8 @@ const Blog: React.FC = () => {
             </Container>
           </div>
           <div className={classes.sectionDiv}>
+          {filteredAuthorData[0] ?
```

Comment:
add this check to line no 197, so we can just display the 404 without any modification for the blog page.

Reply:
This is done intentionally, because consider the below scenarios

Scenario A
when a user internally or externally lands on some undefined URL, instead showing just the 404 page with a button to redirect to blogs, 


Scenario B
Instead we have a the 404 page with blogs button and also some recommended blogs and newsletter sections, this way the users will have much things to work around.

Mainly, the intention was to keep the users stay on the site for longer duration.

cc  @rakeshPRaghu  let me know your thoughts.